js/modal.js

```diff
@@ -14,12 +14,14 @@
   // ======================
 
   var Modal = function (element, options) {
-    this.options        = options
-    this.$body          = $(document.body)
-    this.$element       = $(element)
-    this.$backdrop      =
-    this.isShown        = null
-    this.scrollbarWidth = 0
+    this.options             = options
+    this.$body               = $(document.body)
+    this.$element            = $(element)
+    this.$dialog             = this.$element.find('.modal-dialog')
+    this.$backdrop           =
```

Comment:
FWIW, [other files use the same style](https://github.com/twbs/bootstrap/blob/804457c474c686d4b5f45545da0dbad5ed56e387/js/carousel.js#L20), not that I'm a fan of that style. I'd just repeat `null` if this was my code.